scripts/profiler/plot_nordic.py

```diff
@@ -218,9 +215,9 @@ def _find_closest_event(self, x_coord, y_coord):
                 return None
             matching_processing = list(
                 filter(
-                    lambda x: x.proc_start_time < x_coord and x.proc_end_time > x_coord,
+                    lambda x: x.proc_start_time < x_coord < x.proc_end_time,
                     filtered_id))
-            if len(matching_processing):
+            if len(matching_processing) != 0:
```

Comment:
What linter error check flagged this? I think we should just disable it instead.

Reply:
I guess it was this one:
`len-as-condition (C1801): Do not use len(SEQUENCE) as condition value. Used when Pylint detects incorrect use of len(sequence) inside conditions.`

Reply:
OK. Looking at this again, I think the right way to address this linter complaint is:

```
if matching_processing:
```

Since a nonempty list is truthy, and an empty one is falsy.

Reply:
done